src/buffer/out/textBuffer.cpp

```diff
@@ -944,6 +944,118 @@ Microsoft::Console::Render::IRenderTarget& TextBuffer::GetRenderTarget() noexcep
     return _renderTarget;
 }
 
+// Method Description:
+// - Get the COORD for the beginning of the word you are on
+// Arguments:
+// - target - a COORD on the word you are currently on
+// - wordDelimiters - what characters are we considering for the separation of words
+// - includeCharacterRun - include the character run located at the beginning of the word

```

Comment:
I don't totally get what this is. This just seems like "If we end on a standard character, also select the word zone before it" .. so `GetWordStart(..., true)` is just `GetTheBeginningOfATwoWordRange()`?

Reply:
Okay, so, having read the rest of the code: I saw one person call this with `true`. that person can just run `GetWordEnd` twice...right?

Reply:
The trick is delimiter runs. `includeCharacterRun` is particularly useful in the following case:
```
cool....words
|   | ^target
|   ^ includeCharacterRun = false
^ includeCharacterRun = true
```
That way, you're encompassing the read-able word, not just the "word" you are currently on.